public/app/plugins/datasource/elasticsearch/configuration/ElasticDetails.tsx

```diff
@@ -145,11 +148,11 @@ const jsonDataSwitchChangeHandler =
 
 const intervalHandler =
   (value: Props['value'], onChange: Props['onChange']) => (option: SelectableValue<Interval | 'none'>) => {
-    const { database } = value;
+    const { jsonData } = value;
     // If option value is undefined it will send its label instead so we have to convert made up value to undefined here.
     const newInterval = option.value === 'none' ? undefined : option.value;
 
-    if (!database || database.length === 0 || database.startsWith('[logstash-]')) {
+    if (!jsonData.index || jsonData.index.length === 0 || jsonData.index.startsWith('[logstash-]')) {
```

Comment:
unfortunately, there still may be value in the old database-field.
so i recommend changing this to something like:
```js
const currentIndex = value.jsonData.index ?? value.database;
if (!currentIndex || currentIndex.length === 0 || currentIndex.startsWith('[logstash-]')) {
```